test/test_chain_id_cli_params.py

```diff
@@ -0,0 +1,36 @@
+"""Testing chain id CLI params"""
+
+import subprocess
+
+import pytest
+from starkware.starknet.definitions.general_config import StarknetChainId
+
+from .util import DevnetBackgroundProc, read_stream
+
+ACTIVE_DEVNET = DevnetBackgroundProc()
+
+@pytest.mark.parametrize(
+    "chain_id",
+    [member.name for member in StarknetChainId],
+)
+def test_chain_id_valid(chain_id):
+    """Test if chain id works"""
+    proc = ACTIVE_DEVNET.start(
+        "--chain-id",
+        chain_id,
+    )
+    assert proc.returncode is None
+
+def test_chain_id_invalid():
+    """Test if the invalid chain id fails"""
+    proc = ACTIVE_DEVNET.start(
+        "--chain-id",
+        "",
```

Comment:
This test can be parameterized to include multiple invalid values

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/358/commits/d5fbba57f799eb6e3543663f7511296aaa55fb2a